### ISSUE TITLE
Cannot follow Bookwyrm from Kbin

### ISSUE THREAD

**Issue by chris-y** [bookwyrm-social/bookwyrm#2886]
**Describe the bug**
I'm unable to follow a Bookwyrm account from Kbin

**To Reproduce**
1. Go to Kbin
2. Click on Search and search for a Bookwyrm account
3. Click on Follow
4. Go to Bookwyrm and note that there is no notification of a new follower, nor do they show under followers in the profile

**Expected behavior**
Bookwyrm should recognise when it is being followed by a Kbin account

**Screenshots**
Kbin thinks it is following the account:
![Screenshot_20230630-233552-776.png](https://github.com/bookwyrm-social/bookwyrm/assets/1818803/98a5e218-2e7b-4777-a4f0-32f789cfa7e6)

Can't really show the nonexistence of the follower on the Bookwyrm side, but it's not there!

**Instance**
wyrms.de

**Additional context**
I thought this might have been fixed by #2813 but it appears not. (wyrms.de is on 0.6.3 which has that fix)

**Comment by mouse-reeve** [comment]
Is there a kbin server I can get an account on to test from? I think we'll need to see the logs on BookWyrm to get a clue as to why the follow isn't being created

**Comment by chris-y** [comment]
I think registrations are open on both kbin.social and fedia.io

**Comment by chris-y** [comment]
I think registrations are open on both kbin.social and fedia.io

**Comment by hughrun** [comment]
@chris-y can you or your server admin please try the temporary fix in #2844 and see if it makes a difference?

**Comment by chris-y** [comment]
> @chris-y can you or your server admin please try the temporary fix in #2844 and see if it makes a difference?

Paging @tofuwabohu - are you able to implement this on wyrms.de, then I'll check if it fixes it?

**Comment by tofuwabohu** [comment]
Sorry, I missed the ping.
I stopped the web container and tried `docker-compose run web /app/manage.py runserver 0.0.0.0:8000` which led to 502 errors so I changed it back.

**Comment by hughrun** [comment]
@tofuwabohu you're running on Docker? Can you try adding `--threads=8` to line 50 in your `docker-compose` file as per #2958 ?

**Comment by tofuwabohu** [comment]
Yes, we're running Docker. I added the thread parameter, stopped the web container and ran `docker compose up --build -d`. Should be running threaded now.

**Comment by hughrun** [comment]
@chris-y can you please try kbin again and see what happens?

**Comment by chris-y** [comment]
I unfollowed and refollowed but Bookwyrm is still not acknowledging the follow.

**Comment by hughrun** [comment]
Ok thanks for checking that, it helps troubleshooting.
